ui/demo/less/reader.less

```diff
@@ -58,36 +58,6 @@ body {
   position: absolute;
 }
 
-
-.reader__popover__bbox {
-  border: 1px transparent dotted;
-
-  &.rotate0 {
-    border-bottom-color: @dark-grey
-  }
-  
-  &.rotate90 {
-    border-left-color: @dark-grey;
-  }
-
-  &.rotate180 {
-    border-top-color: @dark-grey;
-  }
-
-  &.rotate270 {
-    border-right-color: @dark-grey;
-  }
-
-  &:hover {
-    background-color: rgba(19, 92, 202, 0.2);
-    cursor: pointer;
-  }
-
-  &.selected {
-    background-color: rgba(19, 92, 202, 0.3);
-  }
-}
-
```

Comment:
Nice, it has to feel good to remove 30 lines of unneeded code 😎

Reply:
@carolinepaulic its not every day we have this chance xD